tags/context.go

```diff
@@ -1,13 +1,16 @@
 package grpc_ctxtags
 
-import "context"
+import (
+	"context"
+)
 
 type ctxMarker struct{}
 
 var (
 	// ctxMarkerKey is the Context value marker used by *all* logging middleware.
 	// The logging middleware object must interf
 	ctxMarkerKey = &ctxMarker{}
+	DeadlineTag  = "deadline"
```

Comment:
`grpc.request.deadline_s` so it is scoped similarly to the rest of the fields in a context.

Reply:
updating it to `grpc.request.deadline` will be a string of `time.RFC3339` format.